### PR TITLE
resize-able manga & chapters list

### PR DESCRIPTION
**Pull request by Robonau**
only realised i wanted it when i used #2347
could be added to that in the same way

dosnt work properly when you disable reader
(works fine after a refresh)

### CODE REVIEW

File: src/web/lib/hakuneko/frontend@classic-dark/app.html [hunk 1]
```diff
@@ -60,12 +62,16 @@
...
+                resize: horizontal;
+                overflow: auto;
             }
             hakuneko-chapters {
                 box-sizing: border-box;
+                flex-grow: 1;
             }
             hakuneko-jobs {
                 flex: 0;
+                width: -webkit-fill-available;
```
Comment (Sheepux) [inline src/web/lib/hakuneko/frontend@classic-dark/app.html]: do not use browser specific css prefix, prefer standard fill-available

File: src/web/lib/hakuneko/frontend@classic-light/app.html [hunk 2]
```diff
@@ -61,12 +63,16 @@
...
+                resize: horizontal;
+                overflow: auto;
             }
             hakuneko-chapters {
                 box-sizing: border-box;
+                flex-grow: 1;
             }
             hakuneko-jobs {
                 flex: 0;
+                width: -webkit-fill-available;
```
Comment (Sheepux) [inline src/web/lib/hakuneko/frontend@classic-light/app.html]: do not use browser specific css prefix, prefer standard fill-available

Comment (Robonau) [reply src/web/lib/hakuneko/frontend@classic-light/app.html]: invalid

**Comment by Sheepux** [comment]
Disabling the reader will ignore de default sizing and therefore creating a blank spacing.
The width should be probably reset tto it's default state in the setting event.
Check in app.html
`             onSettingsSaved(event) {
                this.readerEnabled = Engine.Settings.readerEnabled.value;
            }`
https://github.com/manga-download/hakuneko/blob/master/src/web/lib/hakuneko/frontend%40classic-dark/app.html#L204

**Comment by Sheepux** [comment]
Also, possibly future request: sizing is probably lost on HN restart. Saving it in a local session could allow to restore it on startup.
Also having a "reset to default" option could help.

**Comment by Robonau** [comment]
im not sure how i would save the setting (or even read when it changes) so imma leave it for now

**Comment by Sheepux** [comment]
Finally took time testing it.
Disabling the reader still show the resize option in the manga list, but doesn't work.

**Review by Sheepux — CHANGES_REQUESTED** [review]
As mentioned 1 use case to solve

**Comment by 27136773** [comment]
No activity for some weeks => closed
Feel free to re-open in case of continuing the work

**Pull request closed without merging**